### PR TITLE
Remove usage of java.util.Calendar

### PR DESCRIPTION
**Pull request by RoiEXLab**

### CODE REVIEW

File: src/main/java/games/strategy/triplea/ui/TooltipProperties.java [hunk 1]
```diff
@@ -19,7 +19,7 @@
   private static final String TOOLTIP = "tooltip";
   private static final String UNIT = "unit";
   private static TooltipProperties ttp = null;
-  private static long timestamp = 0;
+  private static Instant timestamp = Instant.now();
```
Comment (ssoloff) [inline src/main/java/games/strategy/triplea/ui/TooltipProperties.java]: Should the right-hand side be `Instant#EPOCH`?

File: src/main/java/games/strategy/triplea/ui/UserActionText.java [hunk 2]
```diff
@@ -17,7 +17,7 @@
   // Filename
   private static final String PROPERTY_FILE = "actionstext.properties";
   private static UserActionText text = null;
-  private static long timestamp = 0;
+  private static Instant timestamp = Instant.now();
```
Comment (ssoloff) [inline src/main/java/games/strategy/triplea/ui/UserActionText.java]: Should the right-hand side be `Instant#EPOCH`?

**Comment by RoiEXLab** [comment]
@ssoloff Updated!

**Comment by RoiEXLab** [comment]
Thanks @ssoloff !

**Pull request merged**